CONTRIBUTING.md

```diff
@@ -13,14 +13,18 @@
 │   ├── common
 │   ├── pages
```

Comment:
Please update this as well, now we don't have any folders in `CSS` directory

Reply:
ok